build.gradle

```diff
@@ -25,6 +25,11 @@ subprojects {
 
   repositories {
     mavenCentral()
+    // This is a workaround to resolve Spark 4.0.0-preview-1
+    // To be removed for official release
```

Comment:
Apache Spark recommends to use `IDed TODO`s. Please file a JIRA issue and use it like the following.

```
- // To be removed for official release
+ // TODO(SPARK-XXXXX) Use Apache Spark 4.0.0-preview1 when it's ready
```